compiler/src/iree/compiler/Reducer/Delta.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Reducer/Delta.h"
+
+using namespace mlir;
+using namespace mlir::iree_compiler;
+
+static FailureOr<WorkItem> checkChunk(Chunk maybeUninterestingChunk,
+                                      Oracle &oracle, WorkItem &root,
+                                      DeltaFunc deltaFunc,
+                                      ArrayRef<Chunk> maybeInterestingChunks,
+                                      DenseSet<Chunk> &uninterestingChunks) {
+  SmallVector<Chunk> currentChunks;
+  copy_if(maybeInterestingChunks, std::back_inserter(currentChunks),
+          [&](Chunk chunk) {
+            return chunk != maybeUninterestingChunk &&
+                   !uninterestingChunks.count(chunk);
+          });
+
+  llvm::errs() << "Checking chunk: \n";
```

Comment:
If you want to keep these logs, you could pass a `llvm::raw_ostream &os` in and send logs to that, rather than use `llvm::errs()` directly. Then, callers could choose what stream to use (if any).

Reply:
I'll add a verbose option for these things I think.